examples/sample.rb

```diff
@@ -0,0 +1,72 @@
+require 'fog'
+require 'fog/kubevirt'
+
+class Kubevirt
+
+  #
+  # Connects to kubevirt
+  # 
+  def connect
+    @conn = Fog::Compute.new(:provider => 'kubevirt', :kubevirt_host => '192.168.121.121', :kubevirt_port => '8443')
```

Comment:
we should include also the kubevitrt token which is mandatory to establish the connection.

Reply:
Correcy, I used `.fog` file. In the example we need a token.